khard/helpers.py

```diff
@@ -168,6 +168,11 @@ def convert_to_yaml(name, value, indentation, index_of_colon,
                             ' ' * (indentation+8), indent_multiline_string(
                                 inner, indentation+12,
                                 show_multi_line_character)))
+            elif isinstance(outer, dict):
+                # ABLABEL'd lists
+                for k in outer:
+                    strings += convert_to_yaml("- " + k, outer[k], indentation+4,
```

Comment:
Can you please fix the indent on this one. It should be one of these
```py
ret = function_name(a, b,
		    c, d)
ret = function_name(
    a, b, c, d)
```
Check [pep8: indent](https://www.python.org/dev/peps/pep-0008/#indentation).